hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEnv.java

```diff
@@ -88,6 +90,13 @@ public HapiTestEnv(@NonNull final String testName, final boolean cluster, final
         }
     }
 
+    // Defines the types of nodes(InProcessHapiTestNode or SubProcessHapiTestNode) for the test executing nodes.
+    enum HapiTestNodesType {
+        // Makes the fist starting node(Alice, id=0) to use InProcessHapiTestNode. This gives us the ability to debug
+        IN_PROCESS_ALICE,
+        UNDEFINED
```

Comment:
This should probably be `OUT_OF_PROCESS_ALICE` for symmetry.  It certainly isn't `UNDEFINED`; it is defined, just not in-process.